Source/Core/DolphinQt/Settings/GeneralPane.cpp

```diff
@@ -147,6 +148,14 @@ void GeneralPane::CreateBasic()
   m_checkbox_cheats = new ConfigBool(tr("Enable Cheats"), Config::MAIN_ENABLE_CHEATS);
   basic_group_layout->addWidget(m_checkbox_cheats);
 
+  m_checkbox_load_games_into_memory =
+      new ConfigBool(tr("Load Games Into Memory"), Config::MAIN_LOAD_GAMES_INTO_MEMORY);
```

Comment:
What do you think about "Load Whole Game Into Memory"?

Reply:
I didn't want to say "whole game", because it does use `DiscScrubber` to only load relevant clusters, but I guess one could argue that scrubbed areas aren't part of the "game".

Hmmmm.. maybe.